examples/vitess/vtctld-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: vtctld
+  labels:
+    name: vtctld
+spec:
+  ports:
+    - port: 15000
+      targetPort: 15000
+      nodePort: 30000
```

Comment:
Is there a way to pick one automatically? Or do you mean just provide a way for the user to pick one and hand it to the script?